src/dredd-command.coffee

```diff
@@ -121,6 +121,12 @@ class DreddCommand
           console.log "  $ gem install dredd_hooks"
         else if config['language'] == 'python'
           console.log "  $ pip install dredd_hooks"
+        else if config['language'] == 'php'
+          console.log "  $ composer require ddelnano/dredd-hooks-php:~1.0.0 --dev"
+        else if config['language'] == 'perl'
+          console.log "  $ cpanm Dredd::Hooks"
+        else if config['language'] == 'go'
+          console.log "  $ go get github.com/snikch/goodman"
```

Comment:
Does this really work? It's not in the README of the project (there are no installation guidelines :( ).